library/sl

```diff
@@ -15,4 +15,4 @@ Tags: 6
 Architectures: amd64
 GitFetch: refs/heads/6
 Directory: sl6
-GitCommit: d2d17b81a784fc91219b6421f15080fb7205eb0d
+GitCommit: 6df01e5a98941ca27585b73c23118879f45d14b7
```

Comment:
Did this build properly?  The tarballs seem to be identical:

```console
$ md5sum \
    <(curl -fsSL 'https://github.com/scientificlinux/sl-docker/raw/d2d17b81a784fc91219b6421f15080fb7205eb0d/sl6/sl-6-docker.tar.xz') \
    <(curl -fsSL 'https://github.com/scientificlinux/sl-docker/raw/6df01e5a98941ca27585b73c23118879f45d14b7/sl6/sl-6-docker.tar.xz')
85b2d07e9325d51b3e6e2eafbf9c537b  /dev/fd/63
85b2d07e9325d51b3e6e2eafbf9c537b  /dev/fd/62
```

Reply:
Weird, I'll rebuild and repush.